### PR TITLE
XSIAM - New Phishing layout + playbook fixes

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Blocked - SDK issue with layout rule

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8729
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8752
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8753
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-8826

## Description
* Created a new layout for phishing in XSIAM
* Fixed attachment upload flow in Process Email - Generic v2
* Fixed an issue where the lack of a default input tag to the Block Indicators - Generic v2 playbook resulted in the absence of blocked indicators in the layout.
* Added playbook trigger recommendation which will run the Phishing - Generic v3 playbook on Phishing alerts.

### CODE REVIEW

**Comment by altmannyarden** [comment]
I don't know if you wanted me to review all files but the layout rule looks great :)

**Comment by idovandijk** [comment]
> I don't know if you wanted me to review all files but the layout rule looks great :)

Only the rule. Still need to fix a few things in other places

**Review by AdiPeret — APPROVED** [review]

**Comment by idovandijk** [comment]
@altmannyarden Can you please review the newly added playbook trigger?
@ShirleyDenkberg Can you please review RN and descriptions?

File: Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md [hunk 1]
```diff
@@ -54,8 +59,10 @@ This playbook does not use any integrations.
...
+| UseOldHTMLFields | This input is used to preserve backward-compatibility. It determines whether the playbook should set email fields that are no longer being used in the out-of-the-box content.<br/>If set to True, the playbook will save data into the the "Email Body HTML" and "Rendered HTML" incident fields as it did before.<br/>If set to False, the playbook will not save data into those fields, and will simply be using the Email HTML field instead.<br/>If you are ingesting large emails which are causing issues with large amounts of data being saved into incident fields, you should set the value to False.<br/>We recommend setting the value to False unless you are certain that you need the "Email Body HTML" and "Rendered HTML" incident fields. | True | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md:62]: Maybe this is none of my business, but...
If we recommend setting this value to False, then shouldn't the default value be False?

File: Packs/Phishing/ReleaseNotes/3_6_0.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
+#### Incident Fields
+
+- **Malicious URL Viewed** - Updated the field's tooltip description.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_6_0.md:4]:
```suggestion
**Malicious URL Viewed** - Updated the field's tooltip description.
```

File: Packs/Phishing/ReleaseNotes/3_6_0.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Layouts
+
+##### New: Phishing Layout
+
+- New: Added a new layout for Phishing alerts. The layout provides 3 tabs for the user to easily find and navigate between the alert details, email details, and the analysis details.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_6_0.md:10]:
```suggestion
New: Added a new layout for Phishing alerts. The layout provides 3 tabs for the user to easily find and navigate between the alert details, email details, and the analysis details.
```

File: Packs/Phishing/ReleaseNotes/3_6_0.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Playbooks
+
+##### Process Email - Generic v2
+
+- Fixed an issue where email attachments were not uploaded to the layout in XSIAM. The playbook now uses the newer Core REST API integration to do that.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_6_0.md:16]:
```suggestion
Fixed an issue where email attachments were not uploaded to the layout in Cortex XSIAM. The playbook now uses the newer Core REST API integration to do that.
```

File: Packs/Phishing/ReleaseNotes/3_6_0.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+
+#### Layout Rules
+
+##### New: Phishing Layout Rule
+
+- New: Added a new layout rule to associate Phishing alerts with the new Phishing layout (Available in Cortex XSIAM).
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_6_0.md:23]:
```suggestion
New: Added a new layout rule to associate Phishing alerts with the new Phishing layout (Available in Cortex XSIAM).
```

File: Packs/Phishing/ReleaseNotes/3_6_0.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+
+#### Triggers Recommendations
+
+- New: **Phishing Playbook Trigger** - Added a trigger recommendation to run the Phishing - Generic v3 playbook on Phishing alerts.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_6_0.md:28]:
```suggestion
New: **Phishing Playbook Trigger** - Added a trigger recommendation to run the Phishing - Generic v3 playbook on Phishing alerts.
```

**Comment by ShirleyDenkberg** [comment]
@AdiPeret @michal-dagan @michalgold @altmannyarden Doc review completed.

File: Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md (hunk 1 above)
Comment (idovandijk) [reply Packs/Phishing/Playbooks/Process_Email_-_Generic_v2_README.md:62]: That's a good observation. However, this is intentional, as we didn't want to break backward-compatibility for existing users, so we recommend them to do it knowingly.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**